solr/server/solr/configsets/_default/conf/solrconfig.xml

```diff
@@ -656,6 +656,10 @@
 
   <!-- Primary search handler, expected by most clients, examples and UI frameworks -->
   <requestHandler name="/select" class="solr.SearchHandler">
+    <shardHandlerFactory name="shardHandlerFactory" class="${shardHandlerFactory:ParallelHttpShardHandlerFactory}">
```

Comment:
I didn't even know this was an option in the solrconfig.xml! Should we make this an option in the solr.xml instead, since that is where it is configured currently?

Reply:
Yep - kindof handy for overriding the solr.xml definition on a per-collection basis.

But I agree - no need to have this in our default configset (even as a commented out example).  Will move it over to solr.xml.

Reply:
I made some minor improvements/refactorings to our ShardHandlerFactory docs, though I think there's still more left undone that I don't want to bite off here.

Will switch the config as well.